### PR TITLE
refactor(frontend): Upgrade to Next.js 11

### PR DESCRIPTION
**Pull request by shelegdmitriy**
Resolves: #667

### CODE REVIEW

File: frontend/next.config.js [hunk 1]
```diff
@@ -53,4 +53,5 @@ module.exports = {
 
     return config;
   },
+  webpack5: false,
```
Comment (shelegdmitriy) [inline frontend/next.config.js:56]: This is a temporary solution to run a project. It must be solved in next iterations

File: frontend/next-env.d.ts [hunk 2]
```diff
@@ -1,10 +1,3 @@
 /// <reference types="next" />
-
-declare module "*.svg" {
-  const content: React.FC;
-  export default content;
-}
-
-declare module "react-text-collapse";
-
-declare module "*"; // for datamaps npm
```
Comment (frol) [inline frontend/next-env.d.ts:10]: I think these might still be necessary (unit tests seem to fail due to these)

**Comment by frol** [comment]
FYI, unit tests should be fixed.

File: frontend/src/libraries/explorer-wamp/index.ts [hunk 3]
```diff
@@ -71,7 +71,7 @@ export class ExplorerApi {
   }
 
   // Establish and handle concurrent requests to establish WAMP connection.
-  static getWampSession(): Promise<autobahn.Session> {
+  static getWampSession(): Promise<autobahn.Session | any> {
```
Comment (shelegdmitriy) [inline frontend/src/libraries/explorer-wamp/index.ts:74]: `| any` - is a temporary solution. As we must to use a new version of `typescript` there is an error: 
Снимок экрана 2021-08-24 в 11 07 12

Thats why I've added a `| any` as a temporary solution. I'll remove it later

**Review by frol — APPROVED** [review]
Cool! I checked it manually on my laptop and it seems to operate just fine.

I will merge it later today.

**Pull request merged**